src/Commands/ProjectVersionCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Terminus Plugin that contain a collection of commands useful during
+ * the build step on a [Pantheon](https://www.pantheon.io) site that uses
+ * a Git PR workflow.
+ *
+ * See README.md for usage information.
+ */
+
+namespace Pantheon\TerminusBuildTools\Commands;
+
+/**
+ * Project Version Command
+ */
+class ProjectVersionCommand extends BuildToolsBase
+{
+    use \Pantheon\TerminusBuildTools\Task\Ssh\Tasks;
+    use \Pantheon\TerminusBuildTools\Task\CI\Tasks;
+
+    /**
+     * Show build info attached to a site created by the
+     * build:project:create command.
+     *
+     * @command build:project:version
+     */
+    public function version()
```

Comment:
Please see https://github.com/consolidation/version-tool/blob/master/src/Cli/InfoCommands.php

Reply:
It would be best to return structured data so that --fields et. al. are available

Reply:
Thanks. I updated accordingly.